cypress/e2e/additems.spec.cy.js

```diff
@@ -0,0 +1,49 @@
+const { baseUrl } = require('../variables/makeup_vars.js');
+
+describe('Добавление товаров в корзину', () => {
+  beforeEach(() => {
+    // Открываем URL маркетплейса
+    cy.visit(baseUrl);
+  });
+
+  it('Проверка открытия маркетплейса', () => {
```

Comment:
Naming. Better to create all the tests name using English

Reply:
Fixed